src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeFromAlive() {
+    Animal.alive = Animal.alive.filter(animal => animal.health !== 0);
```

Comment:
You also can use smth like this:
``` js
removeFromAlive(animal) {
    Animal.alive = Animal.alive.filter(aliveAnimal => aliveAnimal !== animal);
  }
```
Where you compare link of the both objects

Reply:
It sounds more logical to me. This method should remove particular animal from alive, what if you call this method expecting, that it will filter out your desired animal, but some another animal also has health 0?